src/utils.ts

```diff
@@ -2,6 +2,7 @@ import * as childProcess from 'child_process';
 import * as fs from 'fs-extra';
 import * as os from 'os';
 import * as path from 'path';
+import * as semver from 'semver';
```

Comment:
Unused import?

_(Side note: is prettier even run in CI? Why isn't eslint in this repo?? This should get fixed in a separate PR :sob:)_

Reply:
Prettier is run on `test` as well, not sure why it didn't catch it!

From Circle:
```
> @electron/get@0.0.0-development lint /Users/distiller/project
> prettier --check "src/**/*.ts"

Checking formatting...
All matched files use Prettier code style!
```

Reply:
> Prettier is run on `test` as well

:facepalm: I did not scroll in the test output

> not sure why it didn't catch it!

Unused imports isn't a formatting thing, it's a linter thing. Hence my comment about `eslint`.